src/platform/src/Bridge/Ollama/Embeddings.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Platform\Bridge\Ollama;
+
+use Symfony\AI\Platform\Capability;
+use Symfony\AI\Platform\Model;
+
+final class Embeddings extends Model
+{
+    public const NOMIC_EMBED_TEXT = 'nomic-embed-text';
+
+    public const BGE_M3 = 'bge-m3';
+
```

Comment:
```suggestion
    public const BGE_M3 = 'bge-m3';
```

Reply:
Sorry, don't get the suggestion, no extra line?